build-system/global-configs/canary-config.json

```diff
@@ -5,6 +5,6 @@
   "amp-live-list": 1,
   "amp-ios-overflow-x": 1,
   "amp-experiment": 1,
-  "expAdsenseA4A": 0.05,
-  "expDoubleclickA4A": 0.05
+  "expAdsenseA4A": 1.0,
```

Comment:
I don't have a strong preference.  It's a knee-jerk for me to be explicit about floats.  But if you prefer "1", I'm happy to do that.

Made the change and pushed the branch, but I think it will require a new PR, right?